src/command_modules/azure-cli-netappfiles/HISTORY.rst

```diff
@@ -0,0 +1,10 @@
+.. :changelog:
+
+Release History
+===============
+
+
+1.0.0
+++++++++++++++++++
```

Comment:
Should be 2.0.0.  Shouldn't matter much longer, but still.

Reply:
No clear on why? Isn't this the first release of the new command set, so 1.0.0?

Reply:
No, all GA modules in the CLI are 2.0.  Weird historical reasons.